hummingbird/ml/containers/sklearn/onnx_containers.py

```diff
@@ -12,8 +12,10 @@
 import os
 import numpy as np
 import shutil
+import torch
 
-from hummingbird.ml._utils import onnx_runtime_installed, from_strings_to_ints
+import hummingbird
```

Comment:
I see you use this import of `hummingbird` on line 76 to get the versoin...is there additional overhead of always importing the whole thing here at the top vs only importing it if the user calls the `save` function?  (I'm not sure.)  I think it's fine either way

Reply:
Good point. I have no idea. I think that at this point `hummingbird` modules should be already imported, so it is ok?